src/lib/rng/jitter_rng/jitter_rng.cpp

```diff
@@ -0,0 +1,126 @@
+/*
+* CPU Jitter Random Number Generator
+* (C) 2024 Planck Security S.A.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/jitter_rng.h>
+
+extern "C" {
+#include <jitterentropy.h>
+}
+
+namespace {
+
+rand_data* jitter_collector_create();
+void jitter_collector_free(rand_data* collector);
+void jitter_collect_into_buffer(rand_data* collector, std::span<uint8_t> buf);
+
+}  // namespace
+
+namespace Botan {
+
+Jitter_RNG_Data::Jitter_RNG_Data() : m_jitter{jitter_collector_create()} {}
+
+Jitter_RNG_Data::~Jitter_RNG_Data() {
+   jitter_collector_free(m_jitter);
+}
+
+void Jitter_RNG_Data::collect_into_buffer(std::span<uint8_t> buf) {
+   jitter_collect_into_buffer(m_jitter, buf);
+}
+
+std::string Jitter_RNG::name() const {
+   return "JitterRNG";
+}
+
+bool Jitter_RNG::is_seeded() const {
+   return true;
+}
+
+bool Jitter_RNG::accepts_input() const {
+   return false;
+}
+
+void Jitter_RNG::clear() {}
+
+void Jitter_RNG::fill_bytes_with_input(std::span<uint8_t> out, std::span<const uint8_t> in) {
+   m_jitter.collect_into_buffer(out);
+}
+};  // namespace Botan
+
+namespace {
+
+rand_data* jitter_collector_create() {
```

Comment:
At this point these functions can/should be inlined into their respective callers in `Jitter_RNG_Data`

Reply:
Inlined.